other/restrict-annotations/99-delete.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+delete:
+- apiVersion: kyverno.io/v1
+  kind: ClusterPolicy
+  name: require-unique-uid-per-workload
```

Comment:
Copy-and-paste error. The restrict-annotations policy isn't getting removed which is causing failures in the competing "allowed-annotations" test when it is run after this one.

Reply:
Sorry about this! Fixed it!